pkg/agent/agent.go

```diff
@@ -32,6 +37,22 @@ import (
 	"google.golang.org/grpc/status"
 )
 
+type bufferedPipe struct {
+	store catalog.SVIDStores
+	in    pipe.In
+	out   pipe.Out
+}
+
+type bufferedPipes map[string]bufferedPipe
+
+func (p bufferedPipes) pipeIns() map[string]pipe.In {
```

Comment:
I'm not sure if this stuff is here because we are taking pipe size in as a configurable, but I'd really prefer to keep `agent.go` as clean as possible, and tuck this stuff away somewhere else. If this PR is merged as-is, >17% of the `agent.go` code will be handling SVIDStore stuff... compare that with node attestation which is <6%

Reply:
that code is to be able to create 1 Store service for each configured store svid, I'll move it out here.